cmd/driver/install/install_test.go

```diff
@@ -37,6 +37,7 @@ Flags:
       --download                Whether to enable download of prebuilt drivers (default true)
   -h, --help                    help for install
       --http-insecure           Whether you want to allow insecure downloads or not
+      --http-headers string     Optional comma-separated list of headers for the http GET request (e.g. --http-headers='x-emc-namespace: default,Proxy-Authenticate: Basic'). Not necessary if default repo is used
```

Comment:
Here, `--http-headers` should be before `--http-insecure`.

Reply:
Awesome!! Thank you very much! :)